tests/TestCase/View/CellTest.php

```diff
@@ -109,13 +109,8 @@ public function testDebugInfo()
      */
     public function testCellImplictRenderWithError()
     {
-        $capture = function ($errno, $msg) {
-            restore_error_handler();
-            $this->assertSame(E_USER_WARNING, $errno);
-            $this->assertStringContainsString('Could not render cell - Cell template file', $msg);
-        };
-        set_error_handler($capture);
-
+        $this->expectWarning();
+        $this->expectWarningMessage('Could not render cell - Cell template file');
         $cell = $this->View->cell('Articles::teaserList');
         $cell->viewBuilder()->setTemplate('nope');
```

Comment:
Should we move those below this line? To make sure that only the last line fails?

Reply:
Looks like it doesn't matter. I thought we could throw exceptions now, but guess not.